### PR TITLE
Documentation update - Pulse shader

### PR DESCRIPTION
**Pull request by cre8ivepark**
## Overview
Updating pulse shader documentation to match the latest updates.

### CODE REVIEW

File: Documentation/README_PulseShader.md [hunk 1]
```diff
@@ -2,25 +2,37 @@
 
 ![MRTK_SpatialMesh_Pulse](https://user-images.githubusercontent.com/13754172/68261851-3489e200-fff6-11e9-9f6c-5574a7dd8db7.gif)
 
-![MRTK_HandMesh_Pulse2](https://user-images.githubusercontent.com/13754172/68262035-e4f7e600-fff6-11e9-9858-796afd1cabc5.gif)
 Use the pulse shader to animate a visual pulse effect over surface reconstruction, articulated hand mesh, or any other meshes.
 
 ## Shader and material
 
-**MRTK_SurfaceReconstruction.mat** and **MRTK_ArticulatedHandMeshPulse.mat** uses **SR_Triangles** shader. You can configure various options such as fill color, line color, and pulse color.
+Following materials uses **SR_Triangles** shader. You can configure various options such as fill color, line color, and pulse color.
```
Comment (davidkline-ms) [inline Documentation/README_PulseShader.md:9]:
```suggestion
Following materials use **SR_Triangles** shader. You can configure various options such as fill color, line color, and pulse color.
```

File: Documentation/README_PulseShader.md [hunk 2]
```diff
@@ -2,25 +2,37 @@
...
-Use the SurfacePulse.cs script to animate the pulse effect on the assigned material, or turn on "Auto Pulse" in the material itself.
+- **MRTK_Pulse_SpatialMeshBlue.mat** 
+- **MRTK_Pulse_SpatialMeshPurple.mat** 
+- **MRTK_Pulse_ArticulatedHandMeshBlue.mat** 
+- **MRTK_Pulse_ArticulatedHandMeshPurple.mat** 
 
 ## Prerequisites
 
-For surface reconstruction, ensure that MRTK_SurfaceReconstruction.mat is assigned under MRTK Settings -> Spatial Awareness -> Display Settings -> Visible Material.
+For spatial mesh example, ensure that MRTK_Pulse_ArticulatedHandMeshBlue.mat or MRTK_Pulse_ArticulatedHandMeshPurple.mat is assigned under MRTK Settings -> Spatial Awareness -> Display Settings -> Visible Material.
```
Comment (davidkline-ms) [inline Documentation/README_PulseShader.md:18]:
```suggestion
For the spatial mesh example, ensure that MRTK_Pulse_ArticulatedHandMeshBlue.mat or MRTK_Pulse_ArticulatedHandMeshPurple.mat is assigned under MRTK Settings -> Spatial Awareness -> Display Settings -> Visible Material.
```

File: Documentation/README_PulseShader.md [hunk 3]
```diff
@@ -2,25 +2,37 @@
...
 
-For articulated hand, ensure that MRTK_ArticulatedHandMeshPulse.mat is assigned in ArticulatedHandMesh.prefab, which itself should be assigned in MRTK Settings -> Input -> Hand Tracking -> Hand Mesh Prefab.
+For hand mesh example, ensure that MRTK_Pulse_SpatialMeshBlue.mat or MRTK_Pulse_SpatialMeshPurple.mat is assigned in ArticulatedHandMesh.prefab, which itself should be assigned in MRTK Settings -> Input -> Hand Tracking -> Hand Mesh Prefab.
```
Comment (davidkline-ms) [inline Documentation/README_PulseShader.md:20]:
```suggestion
For the hand mesh example, ensure that MRTK_Pulse_SpatialMeshBlue.mat or MRTK_Pulse_SpatialMeshPurple.mat is assigned in ArticulatedHandMesh.prefab, which itself should be assigned in MRTK Settings -> Input -> Hand Tracking -> Hand Mesh Prefab.
```

**Review by davidkline-ms — COMMENTED** [review]
Just a couple minor fixes. Otherwise looks great!

**Review by davidkline-ms — APPROVED** [review]

**Review by CDiaz-MS — APPROVED** [review]

**Pull request merged**